Update font and styling transform a property. 431

AZBL opened the pull request:
This change addresses the following on the Get Access page: 

- Updates copy to match latest prototype
- Adds gray cards for Get Help and Additional Considerations
- Adds links for Detroit Future City vacant property quiz and Park in a Truck (will still need other links)
- Changes links to open in new tabs rather than navigating away from page, which is in line with the rest of the site
- Does not include the green intervention cards

AZBL commented:
@thansidwell @paulhchoi @Moylena  

I'm not exactly sure how my change to global.css to remove what I pasted below got added to this request, but I do believe the plan is to remove that anyway. Let me know if I need to address that or anything else.

 /* Space between search bar and sidepanel, especially for small screens */
.mapboxgl-ctrl-geocoder {
  @apply mt-2;
}

paulhchoi commented on src/app/(content-pages)/transform-property/TransformPropertyPage.tsx:
```diff
@@ -1,20 +1,15 @@
         <h1 className="heading-3xl font-bold mb-6">Transform a Property</h1>
-
-        <p className="body-md">
-          Clean & Green Philly aims to help reduce gun violence by intervening
```
not a breaking issue, but when writing copy, it's always best to replace usages of certain symbols like `&` with `&amp;` — you can read more about HTML entities here: https://developer.mozilla.org/en-US/docs/Glossary/Entity

paulhchoi commented on src/app/globals.css:
```diff
@@ -214,6 +218,14 @@
+  top: 20px;
+  right: 20px;
+}
+
```
I don't believe these css changes are in scope with your PR, please feel free to remove

paulhchoi reviewed (changes requested):
thanks for the work @AZBL! Please make sure to communicate and sync with @Amberroseweeks as her open PR (https://github.com/CodeForPhilly/vacant-lots-proj/pull/514) overlaps quite a bit with yours

Amberroseweeks commented:
@AZBL this page looks really good! I worked on the green cards so I am just taking a look to see if I can align on some of your changes here. It might be best if your changes push first then I can manage any conflicts - focusing on keeping your style changes here.  I will update the links I have to include "rel="noopener noreferrer nofollow"

Could we use the ThemeButtonLink component here on the page? So that we use the same link styling and treatments throughout the pages. You can add the arrow icon through the endContent parameter. Also if we can add an aria label to the links that would be great, for example:

aria-label={link.text + " Link opens in new tab"}

Not sure about the exact wording there @bacitracin can maybe advise on the best phrasing for the label, but please include the link text in the label too.

AZBL commented:
@paulhchoi @Amberroseweeks @thansidwell 
I just made the requested changes. 

- I manually removed all the css changes (honestly not sure what I did to have them included originally in the first place; they were part of another ticket I'm working on)
- changed & to &amp; 
- Added the ThemeButtonLink. I did have to add some className properties to it to get the text color and alignment to match the prototype.
- Also I added an aria-label. @bacitracin could you check if what I added is appropriate?
- Also I added variables for the link urls as I saw that's what was done in other files.
- Finally, for jump start for now I just have it going to "#" as I noticed in the mockup the link for Jump Start goes to Park in a Truck; I'm assuming thats unintentional. 

Please let me know if there are any other changes I need to make.

AZBL commented:
Hey @paulhchoi  could you please advise on how to best resolve the conflicts in globals.css? They involve the lines of code you suggested to be removed. Those lines were part of my PR for issue #475 which was merged into staging and I'm not sure how removing them here would affect staging when this gets removed? Thanks

brandonfcohen1 commented:
> Hey @paulhchoi could you please advise on how to best resolve the conflicts in globals.css? They involve the lines of code you suggested to be removed. Those lines were part of my PR for issue #475 which was merged into staging and I'm not sure how removing them here would affect staging when this gets removed? Thanks

Hey @AZBL, try merging in the latest staging then repushing

AZBL commented:
@brandonfcohen1 @paulhchoi @Moylena

I just resolved the conflicts between @Amberroseweeks and my PRs. I believe now the Transform a Property page looks exactly like the prototype.

A couple things of note: 

- Link for Resources lead to a 404 page. Link to Jump Start just goes to # (reopens the page in a new tab). 

Let me know if there's anything else needed to be done - I won't have access again until this afternoon.

paulhchoi commented on src/app/(content-pages)/transform-property/TransformPropertyPage.tsx:
```diff
@@ -16,11 +17,15 @@ import park from "@/images/install-a-park.jpeg";
 export default function TransformPropertyPage() {
+  const parkInATruckUrl =
+    "https://www.jefferson.edu/academics/colleges-schools-institutes/architecture-and-the-built-environment/programs/landscape-architecture/park-in-a-truck.html";
+  // update below url
+  const jumpStartUrl = "#";
```
after some googling i found this link here, I think we can use it as a placeholder for now until @thansidwell or @nlebovits confirm what other links we should use: https://www.phila.gov/services/payments-assistance-taxes/taxes/tax-credits/business-tax-credits/jump-start-philly/

paulhchoi commented on src/app/(content-pages)/transform-property/TransformPropertyPage.tsx:
```diff
@@ -128,26 +133,32 @@ export default function TransformPropertyPage() {
+          out the
           <a
+            target="_blank"
+            rel="noopener noreferrer nofollow"
             href="https://detroitfuturecity.com/whatwedo/land-use/DFC-lots/"
-            className="link"
+            className="link mx-1"
           >
             Detroit Future City vacant property quiz
-          </a>{" "}
-          (the recommendations work for Philadelphia too!). It's meant to help
```
could we actually leave in the `{" "}` spaces instead of adding a margin? that would actually be the proper way of adding in copy spacing adjacent to components.

paulhchoi commented on src/app/(content-pages)/transform-property/TransformPropertyPage.tsx:
```diff
@@ -275,139 +286,126 @@ export default function TransformPropertyPage() {
+            from local government and non-profit funders, public-private
+            partners, and crowd-sourced fundraising. In the future, we plan to
+            add a page to list common funding sources and explain how to use
+            data from Clean & Green Philly to support a grant application.
```
another ampersand here 😅

paulhchoi commented on src/app/(content-pages)/transform-property/TransformPropertyPage.tsx:
```diff
@@ -275,139 +286,126 @@ export default function TransformPropertyPage() {
+            that may inadvertently increase property values, increase rental
+            prices and displace already-marginalized Philadelphians. We are
+            working to avoid this by meeting extensively with community groups
+            and responding to their concerns. Read more about this topic.
+            {/* Link to gentrification page */}
```
If we don't actually have a link yet, I think we can remove the "Read more..." copy. @thansidwell thoughts?

paulhchoi reviewed (changes requested):
thanks for making the updates! just a few more issues i missed the first go around.

AZBL commented:
@paulhchoi @nlebovits @thansidwell  I'll have access again tomorrow morning to knock these out. 

One more thing. The "Check out our resources to explore more like these" link goes to a 404 page. Do we want to remove that sentence/link until we've created the resources page?

thansidwell commented:
@AZBL Yes, we should leave link that out until we have that page.

thansidwell commented:
@nlebovits Here's the ticket for the Gentrification page. #549

thansidwell commented:
@AZBL @paulhchoi  So from a design standpoint I think everything looks right. 

But it looks like some conflicts need resolving and Paul is requesting some changes. I'll let the two of you work out what needs to get done on that.

AZBL commented:
@nlebovits @thansidwell @Moylena @paulhchoi 

I just resolved the merge conflicts and made Paul's requested changes.

-This also adds in the proper Jump Start link.
-It removes the "check out our resources" copy and link as that was going to a 404 page.
-Also, I thought it made the most sense to just remove the copy about going to the gentrification page as that ticket (#549) hasn't been addressed yet.  Once that page is created (which I'm happy to do) a new ticket can be made for adding back the gentrification link, which should be super simple to do. 

Thanks for your patience with this.
